src/Splat.Common.Test/IDummyInterface.cs

```diff
@@ -8,7 +8,7 @@ namespace Splat.Tests.Mocks
     /// <summary>
     /// A dummy interface used during Locator testing.
     /// </summary>
-    internal interface IDummyInterface
+    public interface IDummyInterface
```

Comment:
Feel free to SuppressMessageAttribute on these interfaces to avoid the warning.

Reply:
done